src/software/ai/hl/stp/play/kickoff_enemy_play_test.cpp

```diff
@@ -28,18 +32,59 @@ TEST_F(KickoffEnemyPlayTest, test_kickoff_enemy_play)
     setRefereeCommand(RefereeCommand::NORMAL_START, RefereeCommand::PREPARE_KICKOFF_THEM);
 
     std::vector<ValidationFunction> terminating_validation_functions = {
-        // This will keep the test running for 9.5 seconds to give everything enough
-        // time to settle into position and be observed with the FullSystemGUI
-        // TODO: Implement proper validation
-        // https://github.com/UBC-Thunderbots/Software/issues/1397
         [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
-            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(9.5))
+            // TODO: Fix bug with robot three not shadowing the enemy kicker in
+            // kickoff_enemy_play
+            // https://github.com/UBC-Thunderbots/Software/issues/1945
+
+            // Friendly robots in position to shadow enemy robots
+            Rectangle robotOneShadowingRect(Point(0, 2.2), Point(-0.4, 1.8));
```

Comment:
Just curious how did you find these points?

Reply:
I made [this Desmos file](https://www.desmos.com/calculator/6trluianaa) for the position of the enemy robot (taken from the test initialization) relative to the field, and roughly estimated the area which the friendly robot should be within so it is shadowing the enemy robot (standing in their way and our goal post)

Reply:
you could explain a bit more where these rectangles are in relation to the ball and the enemy robot?

Reply:
I've updated the explanation to have more detail about the positioning of the rectangles. Feel free to let me know if I've missed anything.